ops/buildkite-pipeline.yaml

```diff
@@ -1,22 +1,43 @@
 steps:
 
 - label: "Developer Environment Build and Test"
+  key: build
   command: |
     echo HOME=$HOME
     echo PATH=$PATH
     nix develop --command just
     nix develop --command just prereqs
     nix develop --command just test
 
-- label: "Nix-Build and Create OCI Image"
+- label: "Tier 2 Testing"
+  key: tier2
+  if: build.branch == "main" && build.env('GRANOLA_NIGHTLY_BUILD') == "true"
+  timeout_in_minutes: 2880
+  depends_on:
+  - build
+  agents:
+    mina-log-storage: true
+    docker: true
+  command: |
+    nix develop --command just build
+    nix develop --command tests/regression test_many_blocks
+    # TODO: why is this not working?
+    # nix develop --command tests/regression test_release
```

Comment:
Did you want to remove this before merging?

Reply:
Is it okay to keep it? It documents the fact that a test is failing.